src/layouts/EditPage.jsx

```diff
@@ -162,12 +191,9 @@ const EditPage = ({ match }) => {
                   <br />
                   <Code>{i + 1}</Code>:
                   <Code>
-                    {elem.attribute?.textContent || elem.element?.textContent
-                      ? (
-                          elem.attribute?.textContent ||
-                          elem.element?.textContent
-                        ).replace("<", "&lt;")
-                      : elem}
+                    {elem.attribute?.nodeName ||
+                      elem.element?.outerHTML ||
+                      elem}
```

Comment:
could i ask why this change was made? i think i might be lacking abit of context here

Reply:
This case assumes that the `textContent` attribute contains a non-null string. However, 2 examples where this can fail:
1. `<script></script>` which will be removed with `element` will have a null `textContent`
2. `<script async>...</script>` which will be removed with `attribute` will have a null `textContent`

Hence, for 1, we get the outerHTML which outputs the entire element that we will be removing, which also provides more context to the user on the text of the whole element that will be removed. For 2, the node name will be "async", which also lets the user know exactly the attributes that will be removed.